### PR TITLE
chore: set default PSM fees to 0%

### PR DESCRIPTION
**Pull request by dckc**
refs: inter xfn meeting yesterday

## Description

Default PSM fees are supposed to be 0 per @rowgraus , @btulloh , @dtribble 

### Security Considerations

none

### Documentation Considerations

@otoole-brendan I suppose this affects screenshots and such

### Testing Considerations

@arirubinstein noted that we have yet to test starting with this configuration.

### CODE REVIEW

File: packages/smart-wallet/test/test-psm-integration.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ test('want stable', async t => {
   await offersFacet.executeOffer(offerSpec);
   await eventLoopIteration();
   t.is(purseBalance(computedState, anchor.brand), 0n);
-  t.is(purseBalance(computedState, stableBrand), swapSize - 1n);
+  t.is(purseBalance(computedState, stableBrand), swapSize); // assume 0% fee
```
Comment (turadg) [inline packages/smart-wallet/test/test-psm-integration.js:174]: confirmed that we have fee math tested in `test-psm.js`

**Review by turadg — APPROVED** [review]
Not sure why this is draft but LGTM.

**Review by dtribble — APPROVED** [review]
LGTM

**Comment by warner** [comment]
Added force:integration to get a head start on the CI

**Comment by dckc** [comment]
Since I wrote it via the web UIi without testing, I wanted to let the ci bots check it before taking it out of draft.

Thanks for review!

**Pull request merged**